content/docs/user-guide/setup-google-drive-remote.md

```diff
@@ -1,9 +1,103 @@
 # Setup a Google Drive DVC Remote
 
-Follow this guide to setup Google Drive as your <abbr>DVC project</abbr>'s
-[remote storage](/doc/command-reference/remote).
+In this guide we explain the existing ways to setup Google Drive
+[remote storage](/doc/command-reference/remote) for your <abbr>DVC
+projects</abbr>, along with the different benefits each one brings.
 
-## Configure a Google Cloud project
+DVC uses the Google Drive API to synchronize your DVC project data with this
+type of remote storage, so it's subject to certain usage limits and quotas,
+which by default are shared with other GDrive remote storage users. For heavy
+use, it's highly recommended to
+[connect using a custom Google Cloud project](#using-a-custom-google-cloud-project),
+which puts you in control of these limits.
+
+Having your own GC project, it's also possible to
+[use a service account](#using-service-accounts) for automating tasks that need
+to establish GDrive remote connections (e.g. CI/CD).
+
+## Quick start
+
+To start using a Google Drive remote, you only need to add it with a
+[valid URL format](#url-format). Then use any DVC command that needs it (e.g.
+`dvc pull`, `dvc fetch`, `dvc push`). For example:
+
+```dvc
+$ dvc remote add -d gdremote gdrive://0AIac4JZqHhKmUk9PDA/dvcstore
+$ dvc push
+
+Go to the following link in your browser:
+
+    https://accounts.google.com/o/oauth2/auth # ... copy this link
+
+Enter verification code: # <- enter resulting code
+```
+
+See [Authorization](#authorization) for more details.
+
+## URL format
+
+There's a few alternatives to construct a GDrive remote URL for different uses,
+such as a folder or subfolder in root, shared folders not owned by your account,
+etc. The URL is formed with a _base_, and an optional _path_ to an **existing**
+folder i.e. `gdrive://<base>/path/to/folder`. The base can be one of:
+
+1. _Folder ID_ - unique identifier for every Google Drive folder, including
+   [shared folders](https://support.google.com/drive/answer/7166529) and
+   [shared drives](https://support.google.com/a/users/answer/9310351) (these two
+   can only be referenced by ID).
+
+   ```dvc
+   $ dvc remote add gdfolder gdrive://0AIac4JZqHhKmUk9PDA
```

Comment:
should we add `-d` here and in other places?

Reply:
I intentionally only added it in the Quick start section where it's follower by `dvc push`. I don't think that it adds value in these other examples.

Reply:
It's not obvious to be honest, if we use `-d` in all other places, let's use it everywhere?

Also, still don't see any value in these special names (one place is left).

Reply:
Names updated now. I accidentally missed those, thanks.

I would rather remove -d from everywhere? The topic of default remotes seems totally separate fom gdrive to me. But then if I remove it from the Quick start example, I then need to add it to `dvc push -r myremote`... Will do that for now... ⏳

Reply:
let's def not do `dvc push -r`, please ... better to include `-d` everywhere ... or even keep as is for now

Reply:
I changed it to dvc remote add --default just in Quick start for now. That way it's more obvious and no need to explain -d or add it everywhere in this doc?

I noticed the dvc push cmd ref examples need an update and I already started another regular updates PR so I can address it there 🙂

Reply:
(See #1110)